scaler/main.go

```diff
@@ -161,85 +140,44 @@ func startGrpcServer(
 	lggr logr.Logger,
 	port int,
 	pinger *queuePinger,
-	routingTable *routing.Table,
+	httpsoInformer informershttpv1alpha1.HTTPScaledObjectInformer,
 	targetPendingRequests int64,
 	targetPendingRequestsInterceptor int64,
 ) error {
 	addr := fmt.Sprintf("0.0.0.0:%d", port)
 	lggr.Info("starting grpc server", "address", addr)
+
 	lis, err := net.Listen("tcp", addr)
 	if err != nil {
 		return err
 	}
 
 	grpcServer := grpc.NewServer()
+	reflection.Register(grpcServer)
```

Comment:
Does the reflection make sense? (just asking, I agree with enabling it)

Reply:
I like to have it enabled because it allows the use of the gRPC CLI and other tools that require this service.